### PR TITLE
Out with the old and in with the new!

### PR DESCRIPTION
**Pull request by ninjacha**
Replace voting panel with html_interface module version ported from
/vg/.

### CODE REVIEW

**Comment by caelaislinn** [comment]
I'd rather not diverge from Baycode significantly, and I don't have the time or inclination to look over this properly. A lot of interfaces had to be changed to use the new module and this is only going to make getting B12 commits almost pointlessly harder.

**Comment by ninjacha** [comment]
But people arent updating the console interfaces anytime soon though

**Comment by ninjacha** [comment]
and I only changed the voting panel
I haven't touched the consoles yet.

File: code/modules/html_interface/cards/cards.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Used by playing cards; /obj/item/weapon/hand
+// Subtype exists because of sendResources; these must be sent when the client connects.
+
+/datum/html_interface/cards/New()
```
Comment (Chnkr) [inline code/modules/html_interface/cards/cards.dm]: So, you also add playing cards? Good job on properly describing the pullrequest.

**Comment by caelaislinn** [comment]
Travis failed on a build error. That should have been fixed before this was merged.

**Pull request merged**